### DISCUSSION TITLE
【词云】词数量比较少时，如何让词与词之间的距离更靠近？

### DISCUSSION THREAD

**Discussion by D15OS** [antvis/G2#6301] [Q&A]
词云数量少的时候，词的布局像是要铺满整个容器一样，导致词的分布很疏
是否可以通过配置让词的布局紧凑居中呢

**Comment by ChristmasFox** [comment]
我也有同样的问题

**Reply by hustcc** [reply]
有具体案例代码吗？是不是字体设置比较小，然后词有比较少，这个就无解了，要么就是设置字体大小设置大一些。

**Reply by D15OS** [reply]
![image](https://github.com/antvis/G2/assets/69630305/6f92ae73-41f3-4543-abe7-cbbf9fe79d34)
就是这样,把demo的数据减少一点
